src/action/app-storage.js

```diff
@@ -26,6 +26,9 @@ class AppStorage {
         if (typeof this._store.settings[key] !== 'undefined') {
           this._store.settings[key] = state[key];
         }
+        if (key === 'restoring') {
+          this._store.settings[key] = JSON.parse(state[key]);
+        }
```

Comment:
Otherwise the property is parsed as a string, so this "casts" it to a boolean.

Reply:
Is there a better way? :grimacing:

Reply:
> Otherwise the property is parsed as a string, so this "casts" it to a boolean.

Really? Why was that not an issue with `store.settings.displayFiat`? I mean the JSON is already parse in line 24? Or am I missing something?